resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackendSuite.scala

```diff
@@ -63,9 +63,15 @@ class KubernetesClusterSchedulerBackendSuite extends SparkFunSuite with BeforeAn
   @Mock
   private var podOperations: PODS = _
 
+  @Mock
+  private var configMapsOperations: CONFIG_MAPS = _
+
```

Comment:
Shall we move this after `labledPods` because it would be better group `PODS and LABELED_PODS` together and `CONFIG_MAPS and LABELED_CONFIG_MAPS` together.